frontend/appflowy_flutter/lib/mobile/presentation/database/field/mobile_field_bottom_sheets.dart

```diff
@@ -26,9 +26,8 @@ void showCreateFieldBottomSheet(
       return DraggableScrollableSheet(
         expand: false,
         snap: true,
-        initialChildSize: 0.97,
-        minChildSize: 0.97,
-        maxChildSize: 0.97,
+        minChildSize: 0.49,
+        maxChildSize: 0.85,
```

Comment:
Just a note here, having this `minChildSize` the same as `initialzChildSize` results in this behavior

https://github.com/AppFlowy-IO/AppFlowy/assets/71320345/3a82b263-f49a-41b6-bd1a-391a8f2b654e

Reply:
how to drag down to close it?

Reply:
If I set it to 0.5, the behavior in the recording happens. If I set it to 0.49, it works.